src/trackinfoobject.cpp

```diff
@@ -37,12 +9,34 @@
 #include "track/keyutils.h"
 #include "util/compatibility.h"
 #include "util/cmdlineargs.h"
-#include "util/time.h"
 #include "util/math.h"
 #include "waveform/waveform.h"
 #include "library/coverartutils.h"
 #include "util/assert.h"
 
+#include <QDomNode>
+#include <QDomDocument>
+#include <QDomElement>
+#include <QFileInfo>
+#include <QDirIterator>
+#include <QFile>
+#include <QMutexLocker>
+#include <QString>
+#include <QtDebug>
+#include <QRegExp>
```

Comment:
Please adhere to the style guide for include order, if everyone makes their own decisions then the guide will not really mean anything :)